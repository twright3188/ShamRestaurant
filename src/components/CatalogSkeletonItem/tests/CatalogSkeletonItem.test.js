import React from 'react';
import renderer from 'react-test-renderer';
import { CatalogSkeletonItem } from '../index';

it('skeleton catalog item renders correctly', () => {
    const tree = renderer.create(<CatalogSkeletonItem />).toJSON();
    expect(tree).toMatchSnapshot();
});