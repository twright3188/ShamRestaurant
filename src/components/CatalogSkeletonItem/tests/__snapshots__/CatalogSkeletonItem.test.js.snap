// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`skeleton catalog item renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 72,
      "justifyContent": "center",
      "padding": 24,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#F2F7F4",
        "height": 24,
        "width": "100%",
      }
    }
  />
</View>
`;
